Specs/Scene/Cesium3DTileBatchTableResourcesSpec.js

```diff
@@ -326,6 +326,14 @@ defineSuite([
             // check that the 2nd resource has an id of 2
             expect(content.getFeature(2).getProperty('id')).toEqual(2);
 
+            // Check that a property can be an array
+            expect(content.getFeature(2).getProperty('rooms')).toEqual(['room2_a', 'room2_b', 'room2_c']);
```

Comment:
Do any of these tests modify with `setProperty` and verify the results?  If not, can we add a new test?